employee-onboarding-A2A-reference-implementation/src/auth/asgardeo.py

```
@@ -0,0 +1,649 @@
+                    client_id,
+                    client_secret,
+                    auth_code, 
+                    pkce.verifier, 
+                    agent_id
+                )
+                vlog(f"\n[STEP 3] ACTOR_TOKEN: {actor_token.token[:20]}...")
+                
+                return actor_token
+        except Exception as e:
+            vlog(f"\n[FATAL] Authorization code flow failed for agent {agent_id}")
+            raise e
```

Ruff (0.15.2)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)